spec/index.bs

```diff
@@ -1786,7 +1788,9 @@ enum GPUFilterMode {
         Select two texels in each dimension and return a linear interpolation between their values. 
 </dl>
 
-{{GPUCompareFunction}} specifies the behavior of a comparison sampler.
+{{GPUCompareFunction}} specifies the behavior of a comparison sampler. If a comparison sampler is
+used in a shader, an input value is compared to the sampled texture value, and the result of this 
```

Comment:
This happens to be much more nuanced that this. First of all, what is the result of comparison? The current text suggests that it's a boolean, when the truth it being a floating point value.
The other complication is that using any filtering with comparison means that the filtering will happen *on top* of the comparison results. I think we need to specify this carefully, possibly in another PR, but need to leave an issue here.

Reply:
How about `0.0 for "fail", and 1.0 for "pass"`?

Reply:
yeah, that would be a good start!

Reply:
please leave an "Issue: describe how filtering interacts with comparison sampling"